### PR TITLE
[Doc] correct format errors in English doc

### PR DESCRIPTION
**Pull request by Sunt-ing**
## Proposed changes
Some formate errors in English doc.
They are very straightforward and should not break any existing build.

### CODE REVIEW

File: docs/en/administrator-guide/broker.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Authentication information is usually provided as a Key-Value in the Property Ma
 
     Simple authentication means that Hadoop configures `hadoop.security.authentication` to` simple`.
 
-    Use system users to access HDFS. Or add in the environment variable started by Broker: ```HADOOP_USER_NAME```。
+    Use system users to access HDFS. Or add in the environment variable started by Broker：```HADOOP_USER_NAME```.
```
Comment (yangzhg) [inline docs/en/administrator-guide/broker.md:167]: should be half width `:` ?

**Comment by Sunt-ing** [comment]
@yangzhg Thanks!

**Comment by Sunt-ing** [comment]
I have updated the PR, please review it again :)

File: docs/en/administrator-guide/http-actions/fe/get-log-file-action.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ None
 
     Specify the log type. The following types are supported:
     
-    * `fe.audit.log`：FE 审计日志
+    * `fe.audit.log`: FE 审计日志
```
Comment (EmmyMiao87) [inline docs/en/administrator-guide/http-actions/fe/get-log-file-action.md:52]:
```suggestion
    * `fe.audit.log`: Audit log of Frontend.
```

File: docs/en/administrator-guide/operation/metadata-operation.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ Adding FE processes is described in detail in the [Deployment and Upgrade Docume
 	* The first time you start a new FE, you must make sure that the `meta_dir` of the FE is created, has correct permissions and is empty.
 	* Starting a new FE and executing the `ALTER SYSTEM ADD FOLLOWER/OBSERVER` statement adds FE to metadata in a sequence that is not required. If a new FE is started first and no statement is executed, the `current node is not added to the group. Please add it first.` in the new FE log. When the statement is executed, it enters the normal process.
 	* Make sure that after the previous FE is added successfully, the next FE is added.
-	* Connect to  MASTER FE and execute `ALTER SYSTEM ADD FOLLOWER/OBSERVER` claus。
+	* Connect to  MASTER FE and execute `ALTER SYSTEM ADD FOLLOWER/OBSERVER` claus.
```
Comment (EmmyMiao87) [inline docs/en/administrator-guide/operation/metadata-operation.md:148]:
```suggestion
	* Connect to  MASTER FE and execute `ALTER SYSTEM ADD FOLLOWER/OBSERVER` stmt.
```

File: docs/en/administrator-guide/operation/metadata-operation.md [hunk 4]
```diff
@@ -239,7 +239,7 @@ FE currently has the following ports
 
 * Ed_log_port: bdbje's communication port
 * http_port: http port, also used to push image
-* rpc_port：FE 的 thrift server port
+* rpc_port: FE 的 thrift server port
```
Comment (EmmyMiao87) [inline docs/en/administrator-guide/operation/metadata-operation.md:242]:
```suggestion
* rpc_port:  thrift server port of Frontend
```

**Comment by Sunt-ing** [comment]
@EmmyMiao87 Thanks!

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**